Remotion/Data/DomainObjects.UnitTests/IntegrationTests/Unload/UnloadDataTest.cs

```diff
@@ -507,7 +507,7 @@ public void UnloadData_ChangedCollection ()
           () => UnloadService.UnloadData(TestableClientTransaction, DomainObjectIDs.OrderItem1),
           Throws.InvalidOperationException
               .With.Message.EqualTo(
-                  "The relations of object 'OrderItem|2f4d42c7-7ffa-490d-bfcd-a9101bbf4e1a|System.Guid' cannot be unloaded.\r\n"
+                  $"The relations of object 'OrderItem|2f4d42c7-7ffa-490d-bfcd-a9101bbf4e1a|System.Guid' cannot be unloaded.{Environment.NewLine}"
```

Comment:
stylistic question:
```suggestion
                  "The relations of object 'OrderItem|2f4d42c7-7ffa-490d-bfcd-a9101bbf4e1a|System.Guid' cannot be unloaded." + Environment.NewLine
```
?

There's a bunch of those tests. Not sure if the string-interpolation syntax to append a trailing newline is the best way to structure this for readability. thoughts and opinions (beside "it's already done now")?

Reply:
Ideally, I would convert this string into a raw string for best readability but some of the lines would be way too long. Comparing the string interpolation and the explicit string concat, I don't see much of a difference in readability. If you look at the code without syntax highlighting, it might be bit more difficult to spot where the interpolations are but that's it. And I'd argue that if the explicit string concat is on the next line it is harder to read so you need to keep it in the same line.

In this case it was easier to fix everything to an interpolation because I could do this with a macro :D

Reply:
Okay.

PS: I'll maintain that it doesn't look like how we used to write code when I was a youngen.